src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -487,6 +487,8 @@ IECore::InternedString g_dispAutoBumpAttributeName( "ai:disp_autobump" );
 IECore::InternedString g_curvesMinPixelWidthAttributeName( "ai:curves:min_pixel_width" );
 IECore::InternedString g_curvesModeAttributeName( "ai:curves:mode" );
 
+IECore::InternedString g_linkedLights( "gaffer:linkedLights" );
```

Comment:
Do you want to store this somewhere more global? We use this in different places...